tests/client.spec.js

```diff
@@ -127,6 +136,13 @@ describe('Client', () => {
 
         expect(parseFloat(httpResponse.grossTotal)).is.a('number')
       })
+
+      it('should have `customerAccountUrl` property', async () => {
+        const httpResponse = await client.issueInvoice(invoice)
+        expect(httpResponse).to.have.property('customerAccountUrl').that.is.satisfies((value) => typeof value === 'string' || value === undefined);
```

Comment:
```suggestion
        expect(httpResponse.customerAccountUrl).to.be.equals('https://www.szamlazz.hu/szamla/fiok/gd82embu556d2qjagzj3s2ijqeqzds4ckhuf')
```

This test ensure the `customerAccountUrl` has been mapped correctly.